src/styles/main.scss

```diff
@@ -1,3 +1,205 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
+
+:root {
+  --card-main: #060b35;
+  --card-secondary: #00acdc;
+  --card-border: #e5e5e5;
+}
+
+html {
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style: none;
+  }
+
+  &__link {
+    height: 60px;
+    white-space: nowrap;
+    display: flex;
+    align-items: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: var(--card-main);
+    font-weight: 500;
+    cursor: pointer;
+
+    &:hover {
+      color: var(--card-secondary);
+    }
+
+    &:active,
+    &--active { /* keeping these for possible tests */
+      position: relative;
+      color: var(--card-secondary);
+    }
+
+    &:active::after,
+    &--active::after { /* keeping these for possible tests */
+      content: "";
+      border-radius: 5px;
+      position: absolute;
+      bottom: 0;
+      left: 0;
+      right: 0;
+      height: 4px;
+      background-color: var(--card-secondary);
+    }
+  }
+
+  &__item + &__item {
+    margin-left: 20px;
+  }
+}
+
+.cards-container {
+  display: flex;
+  flex-wrap: wrap;
+  width: 944px;
```

Comment:
Instead of writing the width with pixels please try to define a function or variable for the width , if we decide to change the spaces between cards everything will be broken then.